src/services/perimeters.ts

```diff
@@ -148,13 +129,13 @@ export const fetchPerimetersInfos = async (perimetersId: string): Promise<Cohort
       monthlyVisitData
     }
   } else if (CONTEXT === 'arkhn') {
-    const services = (await getServices(perimetersId)).filter((service) => undefined !== service.id)
+    const services = (await getOrganizations(perimetersId)).filter((service) => undefined !== service.id)
     const serviceIds = services.map((service) => service.id)
 
-    //FIX: There can be several patients from this request (if a patient is in several services)
     const patientsAndEncountersFromServices = await getPatientsAndEncountersFromServiceId(serviceIds.join(','))
     if (patientsAndEncountersFromServices) {
-      const { patients, encounters } = patientsAndEncountersFromServices
+      const { patients: servicesPatients, encounters } = patientsAndEncountersFromServices
```

Comment:
getPatientsAndEncountersFromServiceId needs refactoring, I think we can fetch patients and encounters in a single api call

Reply:
(and you would not need to use uniqBy)